plugins/modules/grafana_dashboard.py

```diff
@@ -57,6 +57,7 @@
     description:
       - The path to the json file containing the Grafana dashboard to import or export.
       - A http URL is also accepted (since 2.10).
+      - Required if C(state) is C(export) or C(present).
```

Comment:
You can enforce this with the `required_if` parameter in AnsibleModule.
You can add the following (line 507):

```
    module = AnsibleModule(
        argument_spec=argument_spec,
        supports_check_mode=False,
        required_if=[                                  # Addition here
              ['state', 'export', ['path']],           # Addition here
              ['state', 'present', ['path']],          # Addition here
        ],                                             # Addition here
        required_together=[['url_username', 'url_password', 'org_id']],
        mutually_exclusive=[['url_username', 'grafana_api_key'], ['uid', 'slug'], ['path', 'dashboard_id']],
    )
```